meshroom/core/desc.py

```diff
@@ -130,6 +138,11 @@ def matchDescription(self, value, strict=True):
             return self._elementDesc.matchDescription(value[0], strict)
         return True
 
+    elementDesc = Property(Attribute, lambda self: self._elementDesc, constant=True)
+    uid = Property(Variant, lambda self: self.elementDesc.uid, constant=True)
+    joinChar = Property(str, lambda self: self._joinChar, constant=True)
+    instanceType = Property(Variant, getInstanceType, constant=True)
```

Comment:
```suggestion
    instanceType = Property(Variant, lambda self: self.getInstanceType(), constant=True)
```
And then we can remove it from all subclasses.

Reply:
Done.